setup.py

```diff
@@ -4,31 +4,29 @@
 
 __version__ = re.findall(
     r"""__version__ = ["']+([0-9\.]*)["']+""",
-    open("pyxlight/__init__.py").read(),
+    open("cmplxfoil/__init__.py").read(),
 )[0]
 
 this_directory = os.path.abspath(os.path.dirname(__file__))
 with open(os.path.join(this_directory, "README.md"), encoding="utf-8") as f:
     long_description = f.read()
 
 setup(
-    name="pyXLIGHT",
+    name="cmplxfoil",
     version=__version__,
     description="A Python wrapped version of Mark Drela's XFOIL code with the GUI features removed.",
     long_description=long_description,
     long_description_content_type="text/markdown",
     keywords="",
     author="",
     author_email="",
-    url="https://github.com/mdolab/pyXLIGHT",
+    url="https://github.com/mdolab/CMPLXFOIL",
     license="",
     packages=[
-        "pyxlight",
+        "cmplxfoil",
     ],
-    package_data={"pyxlight": ["*.so"]},
-    install_requires=[
-        "numpy",
-    ],
-    extras_require={"testing": ["testflo"]},
+    package_data={"cmplxfoil": ["*.so"]},
+    install_requires=["numpy", "mdolab-baseclasses", "prefoil", "pygeo", "matplotlib"],
```

Comment:
Are all of these _required_? Seems like they might be needed for generating docs or examples, but I don't see them imported otherwise.

Reply:
The first three are (and all available from PyPI), but you're right that `pygeo` is only used in one of the methods, so I'll move the import down there and add it as an optional package. Same for `matplotlib`.

Reply:
Thanks @eytanadler - unfortunately conda packages may only depend on conda packages. Which means if `prefoil`, `pygeo` are required, then they must also be conda packaged. I know that `mdolab-baseclasses` is already packaged for conda, an exercise @nwu63 went through when packaging `pyoptsparse` for conda. Maybe not worth it.

I have a few more thoughts on the conda packaging which I will post at #15

Reply:
It's possible to make this work. I'm going to make `pygeo` optional (only needed to get a triangulated surface mesh, which is usually used to enforce geometric constraints) and remove the `prefoil` dependency by replacing the read/write dat file functions with a new implementation. Would that help?